src/main/java/org/broadinstitute/hellbender/tools/sv/SVCallRecordUtils.java

```diff
@@ -331,6 +321,7 @@ public static SVCallRecord create(final VariantContext variant, boolean keepVari
         final GATKSVVCFConstants.StructuralVariantAnnotationType type = inferStructuralVariantType(variant);
         final GATKSVVCFConstants.ComplexVariantSubtype cpxSubtype =
                 type == GATKSVVCFConstants.StructuralVariantAnnotationType.CPX ? getComplexSubtype(variant) : null;
+        final List<SVCallRecord.ComplexEventInterval> cpxIntervals = parseComplexIntervals(variant.getAttributeAsStringList(GATKSVVCFConstants.CPX_INTERVALS, "."));
```

Comment:
Good catch, a better choice would be "null", although I think in either case it would lead to an error getting thrown. Looking at this had led me to discover that there were some problems with the unit tests (CPX type and intervals were not being checked), which I fixed, and I've added a dictionary check to the complex interval parsing.